### DISCUSSION TITLE
We Built an AI Search Engine that can find exact timestamps for anything on Youtube using OpenAI Whisper

### DISCUSSION THREAD

**Discussion by aarondoerfler** [openai/whisper#927] [Show and tell]
Hi Everyone,

Atlas finds the exact time a topic in a video was mentioned, even if the word itself wasn't said. You can try it at https://atlas.atila.ca/

The code is also open-sourced so you can also run it yourself.

Backend: https://github.com/atilatech/atila-core-service

Frontend: https://github.com/atilatech/atlas-ui

This blog post explains how it works in simple english: https://atila.ca/blog/tomiwa/atlas

![ezgif com-gif-maker](https://user-images.githubusercontent.com/64740620/216850762-096940d9-29c5-4ff7-a516-b85c4f609d6a.gif)

**Comment by rajivpoddar** [comment]
Can it also answer questions about the topic?

**Comment by jwnacnud** [comment]
I get a 500 server error when I try any of the examples.

**Comment by xDerApfelx** [comment]
I get this error code
Error: "\n<!doctype html>\n<html lang=\"en\">\n<head>\n <title>Server Error (500)</title>\n</head>\n<body>\n <h1>Server Error (500)</h1><p></p>\n</body>\n</html>\n"
